specification/monitor/data-plane/Microsoft.Insights/preview/2023-03-01-preview/examples/MetricBatchExample.json

```diff
@@ -0,0 +1,520 @@
+              "type": "Microsoft.Insights/metrics",
+              "name": {
+                "value": "Egress",
+                "localizedValue": "Egress"
+              },
+              "displayDescription": "The amount of egress data. This number includes egress to external client from Azure Storage as well as egress within Azure. As a result, this number does not reflect billable egress.",
+              "unit": "Bytes",
+              "timeseries": [
+                {
+                  "metadatavalues": [
+                    {
+                      "name": {
+                        "value": "apiname",
+                        "localizedValue": "apiname"
+                      },
+                      "value": "GetBlobServiceProperties"
+                    },
+                    {
+                      "name": {
+                        "value": "authentication",
+                        "localizedValue": "authentication"
+                      },
+                      "value": "AccountKey"
+                    }
+                  ],
+                  "data": [
+                    {
+                      "timeStamp": "2020-08-01T20:00:00Z"
+                    },
+                    {
+                      "timeStamp": "2020-08-01T20:15:00Z"
+                    },
+                    {
+                      "timeStamp": "2020-08-01T20:30:00Z"
+                    },
+                    {
+                      "timeStamp": "2020-08-01T20:45:00Z"
+                    },
+                    {
+                      "timeStamp": "2020-08-01T21:00:00Z"
+                    },
+                    {
+                      "timeStamp": "2020-08-01T21:15:00Z"
+                    },
+                    {
+                      "timeStamp": "2020-08-01T21:30:00Z"
+                    },
+                    {
+                      "timeStamp": "2020-08-01T21:45:00Z"
+                    },
+                    {
+                      "timeStamp": "2020-08-01T22:00:00Z",
+                      "average": 880,
+                      "minimum": 880,
+                      "maximum": 880
+                    },
+                    {
+                      "timeStamp": "2020-08-01T22:15:00Z"
+                    },
+                    {
+                      "timeStamp": "2020-08-01T22:30:00Z"
+                    },
+                    {
+                      "timeStamp": "2020-08-01T22:45:00Z"
+                    }
+                  ]
+                }
+              ],
+              "errorCode": "Success"
+            }
+          ],
+          "namespace": "microsoft.storage/storageaccounts",
+          "resourceregion": "eastus"
+        }
+      ]
+    },
+    "default": {
+      "body": {
+        "code": "ExpiredAuthenticationToken",
+        "message": "The access token expiry UTC time '8/6/2020 8:25:30 AM' is earlier than current UTC time '8/6/2020 4:06:04 PM'."
+      }
+    }
```

Comment:
Can you also include an error response that validates the additional info section of the error response such as:

{
  "additionalInfo": [
    {
      "type": "string",
      "info": "TraceId={9f2049bb-bf09-1bbd-2014-20ce9a55ad0f}"
    },
    {
      "type": "string",
      "info": "ExceptionType=Microsoft.Online.Metrics.MetricsMP.Utilities.RPRequestFormatException"
    }
  ],
  "code": "BadRequest",
  "message": "Query endTime: 06/06/2021 00:00:00 is out of the Max metrics retention period: 93.00:00:00, traceId: {9f2049bb-bf09-1bbd-2014-20ce9a55ad0f}"
}

Reply:
Will be fixed in the next commit